api/lib/domain/models/AuthenticationMethod.js

```diff
@@ -62,7 +62,7 @@ const validationSchema = Joi.object({
     .required(),
   authenticationComplement: Joi.when('identityProvider', [
     { is: identityProviders.PIX, then: Joi.object().instance(PixAuthenticationComplement).required() },
-    { is: identityProviders.POLE_EMPLOI, then: Joi.object().instance(PoleEmploiAuthenticationComplement).required() },
+    { is: identityProviders.POLE_EMPLOI, then: Joi.object().instance(OidcAuthenticationComplement).required() },
```

Comment:
**Réflexion** : si l'idee pour la mutualisation c'est qu'il n'y ait plus de référence a un identity provider particulier dans le code a part dans le registry et les différentes sous classes du service oidc, j'ai l'impression qu'ici on aura pas le choix que de les distinguer non ? 🤔

Reply:
on pourra en faire une seule ligne pour tous les identity provider oidc et dans `is` avoir un enum, mais on ne peut pas le faire mnt parce que la CNAV n'a pas encore de complement

Reply:
justement elle ne compte pas en avoir à priori. Donc on pourra pas avoir une seule ligne 😬

Reply:
mais si avec la perennite des comptes, elle en aura besoin

Reply:
Non avec ou sans la pérennité,la cnav n'a rien besoin de stocker en authentication complement. Dispo pour en parler si tu veux